projects/telemetry/src/nmea.c

```diff
@@ -0,0 +1,131 @@
+#include "nmea.h"
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include "log.h"
+#include "status.h"
+#include "util.h"
+
+// Just a function to losely validate if a sentence is valid
+StatusCode is_valid_nmea(const uint8_t *to_check, size_t len) {
+  if (len < 1 && to_check[0] != '$') {
```

Comment:
I don't think the conditions `len < 1` and `to_check[0] != '$'` can ever both be true, since `len < 1` would mean that `to_check[]` has no data. I think replacing the `&&` with `||` might be better here.

Reply:
You're right, thanks for catching that